java/client/src/main/java/glide/api/commands/GenericBaseCommands.java

```diff
@@ -542,4 +544,152 @@ CompletableFuture<Boolean> pexpireAt(
      * }</pre>
      */
     CompletableFuture<Long> touch(String[] keys);
+
+    /**
+     * Sorts the elements in the list, set, or sorted set at <code>key</code> and returns the result.
+     * The <code>sort</code> command can be used to sort elements based on different criteria and
+     * apply transformations on sorted elements.<br>
+     * To store the result into a new key, see {@link GenericBaseCommands#sortStore(String, String)}.
```

Comment:
Optional - when you are referring a method inside the same class, you can completely skip the class name
```suggestion
     * To store the result into a new key, see {@link #sortStore(String, String)}.
```